go/service/ui_router.go

```diff
@@ -155,3 +155,20 @@ func (u *UIRouter) GetRekeyUI() (keybase1.RekeyUIInterface, int, error) {
 	}
 	return ret, sessionID, nil
 }
+
+func (u *UIRouter) GetRekeyUINoSessionID() (keybase1.RekeyUIInterface, error) {
```

Comment:
But @chrisnojima if you'd prefer service to ask for a session id first, i'm fine with that.